remoteappmanager/file_config.py

```diff
@@ -57,6 +64,34 @@ class FileConfig(HasTraits):
         default_value=paths.static_dir,
         help="The path where to search for static files")
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # Sets the default of the docker configuration options from the
+        # current environment. These will possibly be overridded by
+        # the appropriate entries in the configuration file when parse_file
+        # is invoked
+
+        env = os.environ
+
+        self.docker_host = env.get("DOCKER_HOST", "")
+        if self.docker_host == "":
+            self.docker_host = "unix://var/run/docker.sock"
+
+        self.tls_verify = (env.get("DOCKER_TLS_VERIFY", "") != "")
+
+        cert_path = env.get("DOCKER_CERT_PATH", "")
+        if self.tls_verify or cert_path != "":
+            if cert_path == "":
+                cert_path = os.path.join(os.path.expanduser("~"), ".docker")
+
+            self.docker_host = self.docker_host.replace('tcp://', 'https://')
```

Comment:
For the `https://` comment, I see that here now: https://github.com/docker/docker-py/blob/1.9.0-rc2/docker/utils/utils.py#L509

1. Would be good to add a comment here to remind ourselves or others in the future (because it looks like a hack)
2. ContainManager._get_ip_and_port (https://github.com/simphony/simphony-remote/blob/e3045d31926195a81833b1bb2f0f4d04a79b24e5/remoteappmanager/docker/container_manager.py#L392) checks if the hostname starts with `tcp`.  In fact we just take the host after `://` there.